packages/generator-one-app-module/generators/app/index.js

```diff
@@ -63,8 +69,40 @@ module.exports = class extends Generator {
       type: 'input',
       name: 'moduleName',
       validate: helper.validateIfInputIsValidOrNot,
-      message: 'What is the name of your Module?',
-    }];
+      message: 'Name of your Module:',
```

Comment:
Why was this phrasing changed? Seemed friendlier and more inviting before.

Reply:
changed to previous prompt!